bot.py

```diff
@@ -259,6 +264,7 @@ def get_user_info(user: discord.User) -> UnsentMessage:
         message += "\n\n However, `" + username + "` is a valid role. Did you mean `!list " + username + "`?"
     nearest_or_sorry = '", nearest matches:' if top_matches else '", sorry'
     embed = discord.Embed(description=message, title='No users by the exact name "' + username + nearest_or_sorry)
+    embed.set_footer(text='Requested by: '+ctx.message.author.name+'#'+ctx.message.author.discriminator, icon_url=ctx.message.author.avatar_url)
```

Comment:
why not use `.format()` for all of these strings too?

Reply:
Probably a good idea - pity it's not 3.6... f'strings would be even nicer :P  
Most of the previous code just used string +, but will try and stick to .format() for things in future (though this set_footer might end up refactored anyway?)
Since I was just trying to get functionality with this PR, maybe a separate PR going through and updating everywhere to  .format() might be a good idea?